Tests/Container/ContainerTest.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Framework\Tests;
+
+use PHPUnit\Framework\TestCase;
+use Zest\Container\Container;
+
+class ContainerTest extends TestCase
+{
+    public function testContainer()
+    {
+        $container = new Container();
+        $msg = new \Zest\http\Message();
+
+        $instance = $container->register(['Zest\http\Message', 'Message'], $msg, false);
```

Comment:
Once using `use Zest\http\Message`, it can use `Message::class` magic const.

Reply:
Yes, I will apply these changes.